common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -389,16 +431,13 @@
             "off"
         ],
         "getter-return": [
-            "off"
-        ],
-        "guard-for-in": [
             "error"
         ],
-        "id-match": [
+        "guard-for-in": [
             "error"
         ],
         "import/default": [
-            2
+            "off"
```

Comment:
import/default seems like a good one to error on

Reply:
See https://github.com/microsoft/FluidFramework/pull/9748/files#r845682775